packages/components/src/components/Fade.tsx

```diff
@@ -13,16 +13,17 @@ const FadingDiv = styled.div<{ duration: number }>`
 
 export type FadeProps = {
     threshold?: number;
+    children: React.ReactNode;
```

Comment:
nit: Maybe it would make sense to get rid of all `React.FC` in codebase in separate commit to prevent errors with `children`

Reply:
OK, I removed `React.FC` from most of the components in a38f00dcbf532345b27ebf8cc5b64be1ab2189f8. I left it in some RN placeholder components and one HOC.

Reply:
RN placeholder components could be safely removed. They are not used anywhere and won't be.

But doesn't have to by part of this PR for sure.